packages/block-library/src/button/edit.js

```diff
@@ -301,6 +317,8 @@ function ButtonEdit( props ) {
 						...colorProps.style,
 						...spacingProps.style,
 						...shadowProps.style,
+						...typographyProps.style,
```

Comment:
It looks `blockProps` is still adding the style attributes to the wrapper element.

I'm not sure if this is the perfect way to go about it, but we might have to explicitly exclude these keys in the hook arguments above. For example:

```js
	const blockProps = useBlockProps( {
		ref: useMergeRefs( [ setPopoverAnchor, ref ] ),
		onKeyDown,
		style: {
			fontSize: undefined,
			lineHeight: undefined,
			fontFamily: undefined,
			fontWeight: undefined,
			fontStyle: undefined,
			textTransform: undefined,
			textDecoration: undefined,
			letterSpacing: undefined,
		},
	} );
```

Reply:
> It looks like blockProps is still adding the style attributes to the wrapper element.

This might suggest a regression around the skip serialization feature. I'll try and dig a bit deeper later today or tomorrow after I push the block support stabilization changes forward.

Reply:
I've had a quick look at that latest changes and it appears we just need to tweak the properties listed within the `__experimentalSkipSerialization` array. That array doesn't use the `__experimental` prefix for properties. 

If the history here interests, you can find the original PR that allowed skipping individual features in https://github.com/WordPress/gutenberg/pull/36293. An existing example of skipping an experimental feature is the Navigation block and its `textDecoration` even though the support flag for that is still experimental.

Reply:
Thanks! I fixed it 🐛